### PR TITLE
Update iframes.md

### PR DESCRIPTION
**Pull request by reeceatkinson**
### Description

Changed **height: 1000;** to **height: 100vh;**

### Type of Change
- Updating the documentation

### Checklist
- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/html/concepts/iframes/iframes.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The example features an `iframe` element with the URL from the National Park Ser
   class="i-frame-element"
   src="https://www.nps.gov/caco/index.htm"
```
Comment (Dusch4593) [inline content/html/concepts/iframes/iframes.md:28]:
```suggestion
  src="https://www.nps.gov/caco/index.html"
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Hi, @reeceatkinson, this is Brandon from the Free Team. This is an excellent edit because I also believe the example should be responsive! I'm going to request you make one change, however. Instead of making the height responsive with `100vh`, could we keep it at `1000` and set `width` to `100vw`?

```
<iframe
  class="i-frame-element"
  src="https://www.nps.gov/caco/index.html"
  width="100vw"
  height="1000"
>
  Alt-text for IFrame
</iframe>
```

**Comment by Dusch4593** [comment]
@reeceatkinson After you make the changes, we can merge!

**Comment by reeceatkinson** [comment]
All the checks are in progress!

**Review by Dusch4593 — APPROVED** [review]
Great work, @reeceatkinson, thank you! Merged!

**Comment by reeceatkinson** [comment]
Thanks @Dusch4593

**Comment by reeceatkinson** [comment]
When will it be deployed? @Dusch4593

**Comment by Dusch4593** [comment]
> When will it be deployed? @Dusch4593

It should go live on [the Docs site](https://www.codecademy.com/resources/docs/html/iframes) in a week or so. We're always trying to cut down process-time, though. 😄

**Comment by Dusch4593** [comment]
Congrats, @reeceatkinson! 🎉 🎊  The entry is now live: https://www.codecademy.com/resources/docs/html/iframes

Thanks again for your contribution! Looking forward to working with you another time! 😄

**Pull request merged**